pkg/netpol/connlist/connlist.go

```diff
@@ -491,3 +503,8 @@ func (ca *ConnlistAnalyzer) warnBlockedIngress(peerStr string, ingressObjs map[s
 	ca.errors = append(ca.errors, newIngressAnalyzerConnsBlockedWarning(errors.New(warningMsg)))
 	ca.logger.Warnf(warningMsg)
 }
+
+// GetPeersNamesFromDirPath given a dirPath , returns set of names of the peers that were captured in the directory
+func (ca *ConnlistAnalyzer) GetPeersNamesFromDirPath(dirPath string) map[string]bool {
+	return ca.dirToPeersNamesSet[dirPath]
+}
```

Comment:
I think this should not be in a separate function (it assumes that the function `ConnlistFromDirPath` was called before).
Let's changed the returned values of `ConnlistFromDirPath` to include also the list of workloard peers .